app/models/account.rb

```diff
@@ -37,6 +37,17 @@ def self.canonical_cname(cname)
     cname
   end
 
+  # Change the current active Account.
+  # Required until https://github.com/influitive/apartment/pull/307 is merged
+  # @param [String] cname the host name for the account
+  # @example
+  #   Account.use_account!('whatever.somewhere.io')
+  def self.use_account!(cname)
```

Comment:
I'd be happy to make an instance method for the switching stuff, but I didn't want to be confusing since there's already a `switch!` method there. What would I call it `switch_all_the_things!`?